packages/bug-detectors/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "@jazzer.js/bug-detectors",
+	"version": "1.4.0",
+	"description": "Jazzer.js bug detectors",
+	"homepage": "https://github.com/CodeIntelligenceTesting/jazzer.js#readme",
+	"author": "Code Intelligence",
+	"license": "Apache-2.0",
+	"bugs": {
+		"url": "https://github.com/CodeIntelligenceTesting/jazzer.js/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/CodeIntelligenceTesting/jazzer.js.git",
+		"directory": "packages/bug-detectors"
+	},
+	"main": "dist/index.js",
+	"types": "dist/index.d.js",
+	"dependencies": {
+		"@jazzer.js/fuzzer": "*",
+		"@jazzer.js/hooking": "*",
```

Comment:
Is the `hooking` dependency needed?

Reply:
Probably in the future :smile: Deleted for now.